### DISCUSSION TITLE
NPM ping not working with GitHub private registry

### DISCUSSION THREAD

**Discussion by eddiecorrigall** [community/community#14943] [Actions]
Hello, I filed an issue with NPM, but this maybe something that changed in GitHub's API.

https://github.com/npm/cli/issues/4751

Using the ping command to check the GitHub private registry, GitHub API response is "package [ping] was not found".

```bash
$ npm ping --quiet --registry https://npm.pkg.github.com
npm ERR! code E404
npm ERR! 404 Not Found - GET https://npm.pkg.github.com/-/ping?write=true - Package [ping] was not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/eddie/.npm/_logs/2022-04-14T12_58_15_896Z-debug-0.log
```

**Comment by edhends** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
